src/components/dataElement/DataElementOperandSelect.js

```diff
@@ -44,12 +44,16 @@ export class DataElementOperandSelect extends Component {
             style,
         } = this.props;
 
-        if (!dataElementGroup) {
+        let items;
+
+        if (!dataElementOperands && !dataElementGroup && !dataElement) {
             return null;
+        } else if (!dataElementGroup && dataElement) {
+            items = [dataElement];
+        } else {
+            items = dataElementOperands[dataElementGroup.id];
```

Comment:
This might be a little easier to follow as:

```js
if (dataElementOperands && dataElementGroup) {
  items = dataElementOperands[dataElementGroup.id];
} else if (dataElement) {
  items = [dataElement];
} else {
  return null;
}
```

Note that as written int the PR there as a possible bug if `dataElementOperand = null`, `dataElement = null`, and `dataGroup = 'someGroup'` because it will perform `dataElementOperands['someGroup']` which is `null['someGroup']` and an error.

Reply:
Yes, thats's better. Fixed.